lib/constants.rb

```diff
@@ -206,4 +206,17 @@ module Constants
     "22579",        # Pisa San Rossore T.
     "25543",        # I.B.M.
   ]
+
+  # System id defined in a private file
+  # Most of these ids should be there temporary
+  STATIONS_ENABLED_ELSEWHERE = [
```

Comment:
I don't agree, we don't bypass every check, just the one checking for system ids, so I prefer to be precise. Bypassing checks should not be an habbit, so I want to be clear in the intent. In that very case it will be disabled soon.